examples/numeric-aggregate/src/components/DevicePicker/index.tsx

```diff
@@ -18,6 +19,11 @@ export const DevicePicker: FC<IListPicker> = ({
   maxList,
   setEnabled,
 }) => {
+  useEffect(() => {
+    Authentication.waitTilAuthenticated().then((_) => {
+      Authentication.listenForRefresh();
```

Comment:
@jholmes-formant This should refresh the token every hour, My belief is that the token in the custom module expires and if they are navigating from one dashboard to another it will try to refresh with an expired token, which will trigger the server error, which at the same time gets fix by reloading the whole page so that's why maybe customers don't notice it as much.